integration/tests/cook/test_cli.py

```diff
@@ -936,15 +936,18 @@ def test_show_progress_message(self):
 
     def test_show_progress_message_custom_progress_file(self):
         executor = util.get_job_executor_type(self.cook_url)
+        settings = util.settings(self.cook_url)
+        progress_file_env = (util.get_in(settings, 'executor', 'environment', 'EXECUTOR_PROGRESS_OUTPUT_FILE_ENV') or
+                             'EXECUTOR_PROGRESS_OUTPUT_FILE')
```

Comment:
Should we extract a function since we do the same thing in the two tests?

Reply:
Done, introduced `util.retrieve_progress_file_env`